### ISSUE TITLE
v3.0.0-rc8 -> 3.0.0-rc9.3 won't run on Intel Macs (Intel bundle seems missing)

### ISSUE THREAD

**Issue by jacix** [rivafarabi/deckboard#1095]
**Description**
For months my 2016 Macbook Pro (Intel i7) has been running 3.0.0-rc7 very happily.  I tried to upgrade to 3.0.0-rc9.3 but it throws an error because one of the install files is missing Intel bits, then the app comes up grey and nearly blank. Screenshots and full error text are below.  `/var/folders/zd/.../.com.rivafarabi.deckboard.server*` has two relevant files  - one includes x86_64, but the other doesn't:
```
$ file  /var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.MmHG1N 
/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.MmHG1N: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit bundle x86_64] [arm64:Mach-O 64-bit bundle arm64]
/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.MmHG1N (for architecture x86_64):	Mach-O 64-bit bundle x86_64
/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.MmHG1N (for architecture arm64):	Mach-O 64-bit bundle arm64

$ file  /var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY
/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY: Mach-O 64-bit bundle arm64
```

To be thorough I tried rc8, rc9, rc9.1, rc9.2 - all do the same thing, whether from dmg or zip.

3.0.0-rc9.3 runs fine on my M1 Mac.

**To Reproduce**
1. Unpack the zip or dmz file.
2. Run the app.

**Expected behavior**
- It should run as well as 3.0.0-rc7 and earlier.

**Screenshots**
- Attached below, with copy/paste of the error text.

**Server App (please complete the following information):**
 - OS: MacOS 14.4.1
 - Server App: 3.0.0-rc9.3

**Client App (please complete the following information):**
 - OS: iOS 16.7.7s
 - Client App: Deckboard Pro 2.1.0

**Required files**
- `database.db` and `Library/Application\ Support/deckboard/Local\ Storage` directory are in the attached tgz file.

**Additional context**
**Error Text**
```
Uncaught Exception:
Error: Knex: run
$ npm install sqlite3 --save
dlopen(/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY, 0x0001): tried: '/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64h' or 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY' (no such file), '/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64h' or 'x86_64')), '/private/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64h' or 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/private/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY' (no such file), '/private/var/folders/zd/335zf82n4tl0_2dtgdk87nsr0000gn/T/.com.rivafarabi.deckboard.server.69BLwY' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64h' or 'x86_64'))
at Client_SQLite3.initializeDriver (/Applications/Deckboard.app/Contents/Resources/app.asar/node_modules/knex/lib/client.js:202:13)
at new Client (/Applications/Deckboard.app/Contents/Resources/app.asar/node_modules/knex/lib/client.js:83:12)
at new Client_SQLite3 (/Applications/Deckboard.app/Contents/Resources/app.asar/node_modules/knex/lib/dialects/sqlite3/index.js:22:5)
at new knex (/Applications/Deckboard.app/Contents/Resources/app.asar/node_modules/knex/lib/knex-builder/Knex.js:16:28)
at t.connectDatabase (/Applications/Deckboard.app/Contents/Resources/app.asar/dist/main/index.js:2:30310)
at /Applications/Deckboard.app/Contents/Resources/app.asar/dist/main/index.js:2:41821
at suppressedCallback (node:fs:259:5)
at FSReqCallback.oncomplete (node:fs:189:23)
```
![deckboard-3 0 0-rc9 3-error](https://github.com/rivafarabi/deckboard/assets/7086662/fe2842e9-2755-44f1-83e1-ecbf89b3e5eb)
![deckboard-3 0 0-rc9 3-blank-app](https://github.com/rivafarabi/deckboard/assets/7086662/b2b8c7e5-748d-4103-b084-cb0d24ba55fa)
[deckboard-3.0.0-rc9.3_install-bug.tgz](https://github.com/rivafarabi/deckboard/files/15325597/deckboard-3.0.0-rc9.3_install-bug.tgz)

**Comment by rivafarabi** [comment]
This issue has been fixed in version 3.0.0-rc9.4. Though, I have not tested the build on the actual Intel Mac machine, It runs well on my M1 Mac Mini with Rosetta enabled

If you still have this issue, please open another thread. Thank you

**Comment by jacix** [comment]
Worked like a charm.  Thanks for the quick fix!
